src/open_inwoner/scss/admin/_app_overrides.scss

```diff
@@ -8,71 +8,59 @@
 /**
  * DJANGO ADMIN INDEX
  */
-
-.dropdown-menu,
-div#container .dropdown-menu {
-  /* Override for Environment banner */
-  top: 48px;
-  /* Added to attach body */
-  border-bottom: 8px solid $color_secondary;
+body {
+  --djai-tab-bg: #{$color-primary}; // no fallback to default admin vars
+  --djai-tab-bg--active: #{$color-secondary};
+  --djai-tab-bg--hover: #{$color-primary-dark};
+  --djai-tab-fg--active: #{$color-primary};
+  --djai-dropdown-bg--active: var(--color-primary);
+  --djai-dropdown-bg--hover: #{$color-primary};
 }
 
-.breadcrumbs {
-  top: 48px !important;
-}
+$djai-padding: 40px;
+$djai-padding--mobile: 15px;
+$djai-padding--tablet: 30px;
+$djai-border-width: 8px;
 
-/* Integration into color scheme */
-.dropdown-menu {
-  background-color: $color_primary;
+.djai-dropdown-menu {
+  // use full width on mobile & tablet
+  width: 100%;
 
-  .dropdown-menu__item,
-  .dropdown-menu__item:link,
-  .dropdown-menu__item:visited {
-    color: $color_lightest;
+  /* Added to attach to body/breadcrumbs */
+  border-bottom: $djai-border-width solid $color_secondary;
```

Comment:
Not sure if this is the culprit here - but I see an odd border on the bottom here that makes the header/menu jump out strangley.
Do you also see this? I've added the inspector: if you toggle the border off, it looks normal.

<img width="1351" alt="adminmenu1" src="https://github.com/maykinmedia/open-inwoner/assets/118177951/df077814-be54-4c6d-bff1-2e5ccd642545">

Clearer view on left-side only (but it's also happening on the right-hand side):

<img width="970" alt="adminmenu2" src="https://github.com/maykinmedia/open-inwoner/assets/118177951/dac28a6f-aeb2-4d5c-8fe1-30bf64bc6726">

Reply:
Good catch! I removed the border.